src/database.ts

```diff
@@ -2713,7 +2717,10 @@ class Database extends common.GrpcServiceObject {
         ? (optionsOrRunFn as RunTransactionOptions)
         : {};
 
-    const getWriteSession = this.pool_.getWriteSession.bind(this.pool_);
+    options.optimisticLock;
```

Comment:
Not sure what this line over here is supposed to do, did you mean to assign it to something?

Reply:
I forgot to clean up this line. It's removed now. thanks!